interface/wx/grid.h

```diff
@@ -2808,6 +2808,11 @@ class wxGrid : public wxScrolledWindow
     bool IsCellEditControlEnabled() const;
 
     /**
+        Returns @true if the in-place edit control is currently shown.
```

Comment:
This should probably mention what differentiates it from `IsCellEditControlEnabled()`. And maybe reference `HideCellEditControl()`?